docs/hugo/content/guide/authentication/credential-format.md

```diff
@@ -426,6 +426,127 @@ EOF
 {{% /tab %}}
 {{< /tabpane >}}
 
+## Managed Identity
```

Comment:
This title could possibly be confusing because we also support Azure Workload Identity (which is managed identity too).

Maybe we should change the titles here to be:

1. Managed Identity (via workload identity): Also update line 15 to phrase it that way too?
2. Managed Identity (via IMDS on Azure infrastructure)
3. [Deprecated] Managed Identity (via aad-pod-identity)

Reply:
Updated, thanks!